che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/deploy/_new/NewApplicationViewImpl.java

```diff
@@ -343,13 +348,16 @@ public Project getOpenShiftSelectedProject() {
 
     @Override
     public void setProjects(List<Project> projects) {
+        emptyProjectsMessage.setVisible(projects == null || projects.isEmpty());
```

Comment:
You can move (projects != null && !projects.isEmpty()) to boolean variable isProjectAvailable or something like this and use it three times